### PR TITLE
First Batch for Better Transitions

### PR DESCRIPTION
**Pull request by johko**
I worked on the transitions between units a bit and especially the start of the Multimodal unit.
I also added the Organisation Team to the credits :wink: 

Let me know what you think :hugs:

### CODE REVIEW

**Review by ATaylorAerospace — APPROVED** [review]
LGTM!

**Comment by seshupavan** [comment]
The changes made to unit-0 Welcome ( which we are responsible for) looks good and the credits for organizing team fits well. Pretty happy with what've achieved together 🤗

PS - I'm not a reviewer to approve this

**Comment by merveenoyan** [comment]
@johko the table rendering is chef's kiss

File: chapters/en/Unit 0 - Welcome/welcome.mdx [hunk 1]
```diff
@@ -158,4 +157,7 @@ Our goal was to create a computer vision course that is beginner-friendly and th
 - Reviewers: [Ratan Prasad](https://github.com/ratan), [Ameed Taylor](https://github.com/atayloraerospace), [Mohammed Hamdy](https://github.com/mmhamdy)
 - Writers: [Farros Alferro](https://github.com/farrosalferro), [Mohammed Hamdy](https://github.com/mmhamdy), [Louis Ulmer](https://github.com/lulmer), [Dario Wisznewer](https://github.com/dariowsz), [gonzachiar](https://github.com/gonzachiar)
 
+**Organisation Team**
+[Merve Noyan](https://github.com/merveenoyan), [Adam Molnar](https://github.com/lunarflu), [Johannes Kolbe](https://github.com/johko)
```
Comment (merveenoyan) [inline chapters/en/Unit 0 - Welcome/welcome.mdx:161]:
```suggestion
[Merve Noyan](https://github.com/merveenoyan), [Adam Molnar](https://github.com/lunarflu), [Johannes Kolbe](https://github.com/johko)
We'd like to thank [Maria Khalusova](https://huggingface.co/MariaK) for her thorough reviews.
```

**Review by merveenoyan — APPROVED** [review]
Thanks a lot

**Pull request merged**